source/includes/_tasks.md

```diff
@@ -75,9 +75,9 @@ print(response.text)
 > Sample request:
 
 ```shell
-curl -X GET -H "Content-Type: application/json" -H "Cache-Control: no-cache" -H "X-Api-Key: YOUR API KEY HERE" -d '{
+curl -X POST -H "Content-Type: application/json" -H "Cache-Control: no-cache" -H "X-Api-Key: YOUR API KEY HERE" -d '{
```

Comment:
i thought we still wanted to keep it a GET request  but change the data to be sent as query parameters
```
curl -X GET \
     -H "Content-Type: application/json" \
     -H "Cache-Control: no-cache" \
     -H "X-Api-Key: AJyi1UHbAa1psYrITKKC2w" \
     "https://api.apollo.io/v1/tasks/search?sort_by_field=completed_at&open_factor_names[]=task_types"
```